README.md

```diff
@@ -38,6 +38,50 @@ $ php bin/console server:start
 $ open http://localhost:8000/
 ```
 
+### For Docker installation
+
+Running the local environment with containers:
+
+```bash
+$ docker-compose pull
+$ docker-compose build
+$ docker-compose -f docker-compose.yml -f docker/docker-compose.setup.yml run setup
```

Comment:
The `--rm` switch should be added tot he run command in order to remove the created container after finishing the setup. Currently if setup is run multiple times, it will create as many containers, leaving mess behind.